vllm/v1/engine/async_llm.py

```diff
@@ -84,6 +84,7 @@ def __init__(
         Returns:
             None
         """
+        self._startup_ts = time.time()
```

Comment:
Use `time.monotonic()` so the timestamp is unaffected by system clock changes. See https://docs.vllm.ai/en/latest/design/metrics.html#interval-calculations